Mathlib/Analysis/Convex/Mul.lean

```diff
@@ -144,6 +144,7 @@ lemma ConvexOn.pow (hf : ConvexOn 𝕜 s f) (hf₀ : ∀ ⦃x⦄, x ∈ s → 0
 /-- `x^n`, `n : ℕ` is convex on `[0, +∞)` for all `n`. -/
 lemma convexOn_pow : ∀ n, ConvexOn 𝕜 (Ici 0) fun x : 𝕜 ↦ x ^ n :=
   (convexOn_id <| convex_Ici _).pow fun _ ↦ id
+#align convex_on_pow convexOn_pow
```

Comment:
Note to self: these statements were moved here from `Mathlib/Analysis/Convex/SpecificFunctions/Basic.lean`. (Not sure why they were there in the first place!)